### DISCUSSION TITLE
[Projects beta] Support for notecards

### DISCUSSION THREAD

**Discussion by akampen** [community/community#6721] [Projects and Issues]
It would be great if we could add notecards like we can in the current non-beta projects (and make the notecards assignable).

Sometimes there are tasks that don't make sense as an issue in a repo and note cards are a good solution to this.

**Comment by mr13tech** [comment]
Agree, current projects beta notes, are one liners, pasting text with new lines is not processed; 

Also it would be great to extend notes with extra features, like assignee, labels, etc
